src/auth/integration-tests/Cargo.toml

```diff
@@ -24,7 +24,7 @@ run-integration-tests = []
 
 [dependencies]
 auth          = { path = "../../../src/auth", package = "google-cloud-auth" }
-gax           = { path = "../../../src/gax", package = "google-cloud-gax" }
+gax           = { path = "../../../src/gax", package = "google-cloud-gax", features = ["unstable-stream"] }
```

Comment:
This seems like an unwanted change.  If you are not enabling streaming for any of the client libraries used directly, why do you need to enable streaming for the indirect dependency?

Reply:
I enabled the `unstable-stream` feature `gax` anticipating that we would remove the `unstable-stream` feature from the client crates.